### ISSUE TITLE
CMI Throws an error when using /np preview (nameplates plugin)

### ISSUE THREAD

**Issue by Lorenzostuy** [Zrips/CMI#4649]
**Description of issue:**

When using the command /np preview from the nameplates plugin
https://www.spigotmc.org/resources/nameplates-»-completely-unique-customization-mechanic.84797/
it throws an error in console
https://gyazo.com/b88846e4b3ff808088df68310cfdeab9

---

**ERROR (DELETE IF YOU HAVE NO ERROR):**
```
>.... [17:34:59 INFO]: SlothSetups issued server command: /np preview
>.... [17:35:02 INFO]: SlothSetups issued server command: /eglow
>.... [17:35:09 ERROR]: Could not pass event EntityDismountEvent to CMI v8.7.10.8
java.lang.NullPointerException: null
at com.Zrips.CMI.Modules.Animations.AnimationListener.VehicleExitEvent(AnimationListener.java:45) ~[?:?]
at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor798.execute(Unknown Source) ~[?:?]
at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.Entity.removePassenger(Entity.java:2331) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.Entity.stopRiding(Entity.java:2246) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.EntityLiving.stopRiding(EntityLiving.java:2938) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.EntityLiving.stopRiding(EntityLiving.java:2933) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.WorldServer.a(WorldServer.java:1349) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.WorldServer.entityJoinedWorld(WorldServer.java:1291) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.World.a(World.java:898) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.WorldServer.doTick(WorldServer.java:942) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.MinecraftServer.b(MinecraftServer.java:1416) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.DedicatedServer.b(DedicatedServer.java:372) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.MinecraftServer.a(MinecraftServer.java:1267) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:999) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.3.jar:git-Tuinity-"5df4a4f"]
at java.lang.Thread.run(Thread.java:834) [?:?]
```

**CONFIG SECTION (DELETE IF NOT RELEVANT):**
```
#######################################
## REPLACE WITH RELEVANT CONFIG SECTION  ##
#######################################
```

---

**Cmi Version (using`/cmi version`):** 
CMI8.7.10.8

**Server Type (Spigot/Paperspigot/etc):** 
Tuinity 

**Server Version (using `/ver`):** 
git-Tuinity-"5df4a4f" (MC: 1.16.3)
Basically latest version

**Relevant plugins (Delete if this isn't needed):**
nameplates

**Comment by Zrips** [comment]
Should be resolved with next version
